pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
+name = "onionshare"
+version = "2.2"
+description = "OnionShare lets you securely and anonymously send and receive files. It works by starting a web server, making it accessible as a Tor onion service, and generating an unguessable web address so others can download files from you, or upload files to you. It does _not_ require setting up a separate server or using a third party file-sharing service."
+authors = ["Micah Lee <micah.lee@theintercept.com>"]
+license = "GPLv3+"
+
+[tool.poetry.dependencies]
+python = "^3.7"
```

Comment:
Actually sadly there's a bug where onionshare doesn't work with python 3.8 (#1079) though I'd love to fix that. So this should just be "3.7"

Reply:
Or, however you do >= 3.7 and < 3.8

Reply:
Actually, I'm hoping that upgrading to stem 1.8.0 will allow onionshare will work with python 3.8 as well. If that's the case, then this PR will also fix #1079

Reply:
Can do that in this PR itself.